README.md

```diff
@@ -1,11 +1,31 @@
-     ____            _        _     ____            _     _                         _ 
-    |  _ \ ___   ___| | _____| |_  |  _ \  __ _ ___| |__ | |__   ___   __ _ _ __ __| |
-    | |_) / _ \ / __| |/ / _ \ __| | | | |/ _` / __| '_ \| '_ \ / _ \ / _` | '__/ _` |
-    |  _ < (_) | (__|   <  __/ |_  | |_| | (_| \__ \ | | | |_) | (_) | (_| | | | (_| |
-    |_| \_\___/ \___|_|\_\___|\__| |____/ \__,_|___/_| |_|_.__/ \___/ \__,_|_|  \__,_|
-                                                                                   
-   
-   
-## Current version 
-v.0.1.0                                                                                   
+# RocketDashboard
 
+A dashboard with Jira integration.
+It's perfect for development teams who want to be aware of their progress in real time.
+
+## Widgets
+
+- **Bugs History** – bugs amount chart.
+- **Week Number** – the current week number.
+- **Bugs Difference** – contains two numbers: the current amount of bugs and the amount of bugs one week ago.
+- and many more are coming.
+
+
+## Installation
+
+- Clone ([SSH](git@github.com:rocket-internet-berlin/RocketDashboard.git) or [HTTPS](https://github.com/rocket-internet-berlin/RocketDashboard.git)) or [download](https://github.com/rocket-internet-berlin/RocketDashboard/archive/master.zip) the repository.
+
+- [Install](https://nodejs.org) **node.js**.
+
```

Comment:
Please add the install and run steps:
- `node install` or `yarn`
- `npm start` or `yarn start`
- Open `localhost:3000`